src/V3Broken.cpp

```diff
@@ -169,6 +169,7 @@ class BrokenCheckVisitor final : public VNVisitor {
         UASSERT_OBJ(!whyp, nodep,
                     "Broken link in node (or something without maybePointedTo): " << whyp);
         if (nodep->dtypep()) {
+            if (!nodep->dtypep()->brokeExists()) nodep->dtypep()->dumpTree();
```

Comment:
```suggestion
            if (debug() && !nodep->dtypep()->brokeExists()) nodep->dtypep()->dumpTree();
```
Then add to master

Reply:
This is just accidental debug code, will remove instead as that dump often crashes when it doesn't exists.